frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,460 @@
+                    />
+                    <Label
+                      htmlFor="backup2"
+                      className="ml-2 text-sm text-foreground"
+                    >
+                      I understand the recovery phrase AND a backup of my hub
+                      data directory is required to recover funds from my
+                      lightning channels.
+                    </Label>
+                  </div>
+                )}
+                <div className="flex justify-end gap-2 items-center">
+                  <div className="flex justify-center">
+                    <Button type="submit" size="lg">
+                      Finish Backup
+                    </Button>
+                  </div>
+                </div>
+              </form>
+            </DialogContent>
+          </Dialog>
+        </div>
+      )}
+      <>
+        <Separator className="my-6" />
+        <div className="flex flex-col gap-8">
+          <div>
+            <h3 className="text-lg font-medium">Channels Backup</h3>
+            <p className="text-sm text-muted-foreground">
+              Your spending balance is stored in your lightning channels. In
+              case of recovery or migration or your Alby Hub, they need to be
+              backed up every time you open a new channel.
+            </p>
+          </div>
+
+          <div>
+            <div className="flex gap-2 mb-1 items-center">
+              <h3 className="text-sm font-medium">Automated Channels Backup</h3>
+              {info?.albyAccountConnected ? (
+                <Badge variant={"positive"}>Active</Badge>
+              ) : (
+                <Badge>Recommended</Badge>
+              )}
+            </div>
+            {info?.albyAccountConnected ? (
+              <>
+                <p className="text-muted-foreground text-sm mb-8">
+                  Your channel state is backed up automatically after each
+                  channels creation. Potential recovery will trigger channel
+                  closures, and your funds will arrive in your on-chain balance.
+                </p>
+                {info?.vssSupported && (
+                  <>
+                    <div>
+                      <div className="flex gap-2 mb-1 items-center">
+                        <h3 className="text-sm font-medium">
+                          Dynamic Channels Backup With Instant Recovery
+                        </h3>
+                        {me?.subscription.buzz && info.ldkVssEnabled ? (
+                          <Badge variant={"positive"}>Active</Badge>
+                        ) : (
+                          <Badge>Alby Cloud</Badge>
+                        )}
+                      </div>
+                      <p className="text-sm text-muted-foreground mb-4">
+                        When enabled, your channels state is dynamically updated
+                        on a virtual disk encrypted with your wallet keys
```

Comment:
This copy is wrong in 2 parts: "Virtual Disk", and migrate. I don't think we should mention migrating at all here, we have a dedicated migration page.

Suggestion: 
`When enabled, your channels state is dynamically updated and stored end-to-end encrypted by Alby's Versioned Storage Service. This allows you to recover your spending balance with your recovery phrase alone, without having to close your channels.`